.github/workflows/starknet-contracts.yml

```diff
@@ -24,6 +24,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: software-mansion/setup-scarb@v1
+      - name: Scarb version
+        run: scarb --version
+        working-directory: onchain
+      - uses: actions/checkout@v3
+      - uses: foundry-rs/setup-snfoundry@v3
+        with:
```

Comment:
Is it necessary to specify snfoundry version here?

Reply:
yes, else the action will use 0.24.0 (latest),

currently snforge 0.24.0 is throwing buffer error while deploying the contract on this line 

` let (contract_address, _) = contract.deploy(@social_account_calldata).unwrap();`

specifying the version mean action will that version instead of using the latest one, 

and it is ok, we stick with 0.23.0 for now